app/model/editions/PublishedIssue.scala

```diff
@@ -49,8 +49,9 @@ case class PublishedCollection(id: String, name: String, items: List[PublishedAr
 case class PublishedFront(id: String, name: String, collections: List[PublishedCollection], swatch: Swatch)
 
 case class PublishedIssue(
-  id: String,
-  name: String,
+  id: String, // TODO: Not sure we should be leaking our internal ID here...
+  name: String, // TODO: remove this once downstream is consuming 'edition'
+  edition: String,
```

Comment:
minor: feels like we can make `edition` an enum to be stricter.

Reply:
Done. This will definitely have conflicts with #1025. Maybe get that in first and then merge this second.